skill-tree/src/test/java/com/RDS/skilltree/EndorsementsIntegrationTests.java

```diff
@@ -37,7 +37,7 @@ public EndorsementsIntegrationTests(UserService userService, UserRepository user
     }
 
     @BeforeEach
-    private void addData() throws MalformedURLException {
+    public void addData() throws MalformedURLException {
```

Comment:
move all integrations tests in separate folder

Reply:
Moved the integration tests to /integration folder